Modules/cjkcodecs/multibytecodec.c

```diff
@@ -984,6 +1061,7 @@ mbiencoder_dealloc(MultibyteIncrementalEncoderObject *self)
 {
     PyObject_GC_UnTrack(self);
     ERROR_DECREF(self->errors);
+    Py_CLEAR(self->pending);
```

Comment:
I think this is necessary for the case where the encoder gets garbage collected while there are still pending bytes. I'm not sure why it wasn't here before though...